virtcontainers/pkg/oci/utils_test.go

```diff
@@ -701,9 +701,11 @@ func TestAddAgentAnnotations(t *testing.T) {
 			"e1000e InterruptThrottleRate=3000,3000,3000 EEE=1",
 			"i915 enable_ppgtt=0",
 		},
+		ContainerPipeSize: 1024,
 	}
 
 	ocispec.Annotations[vcAnnotations.KernelModules] = strings.Join(expectedAgentConfig.KernelModules, KernelModulesSeparator)
+	ocispec.Annotations[vcAnnotations.AgentContainerPipeSize] = "1024"
```

Comment:
Since annotations are strings, how about adding a test to assert an error if the pipe size is set to something like "foo"?

Reply:
Done!